swagger_spec_validator/validator20.py

```diff
@@ -43,17 +44,51 @@ def validate_ref(ref_dict):
 
     :raises: :py:class:`swagger_spec_validator.SwaggerValidationError`
     """
-
     keys_to_ignore = {'x-scope', '$ref', 'description'}
 
-    if any(key for key in ref_dict.keys() if key not in keys_to_ignore):
-        swagger_validation_warning = SwaggerValidationWarning(
-            'Found "$ref: {}" with siblings that will be overwritten. '
-            'See https://stackoverflow.com/a/48114924 for more information.'.format(
-                ref_dict['$ref']
-            )
+    if any(key for key in iterkeys(ref_dict) if key not in keys_to_ignore):
+        warnings.warn(
+            SwaggerValidationWarning(
+                'Found "$ref: {}" with siblings that will be overwritten. '
+                'See https://stackoverflow.com/a/48114924 for more information. (path {})'.format(
+                    ref_dict['$ref'], '/'.join(path),
+                )
+            ),
         )
-        warnings.warn(swagger_validation_warning)
+
+    if ref_dict['$ref'] is None:
+        warnings.warn(
+            SwaggerValidationWarning(
+                'Identified $ref with None value. This usually represent an error on the specs even '
+                'if this does not make them invalid as the location of the reference could tolerate '
+                'a None value. (path {})'.format('/'.join(path)),
```

Comment:
Identified $ref with None value. This is usually an error, although technically it might be allowed. (path: {})